Lesson2/src/main/java/pl/solvd/university/Main.java

```diff
@@ -0,0 +1,114 @@
+package pl.solvd.university;
+
+import org.apache.commons.io.FileUtils;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import pl.solvd.university.administration.AdmissionsOffice;
+import pl.solvd.university.administration.Deanery;
+import pl.solvd.university.administration.ExamBoard;
+import pl.solvd.university.departments.Speciality;
+import pl.solvd.university.people.Applicant;
+import pl.solvd.university.people.Dean;
+import pl.solvd.university.people.Professor;
+import pl.solvd.university.state.*;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Locale;
+import java.util.Scanner;
+
+import static pl.solvd.university.administration.AdmissionsOffice.applicants;
+import static pl.solvd.university.administration.ExamBoard.MIN_PASS_SCORE;
+import static pl.solvd.university.departments.Faculty.*;
+import static pl.solvd.university.documents.ExamSheet.grades;
+
+public class Main {
+    public static final Logger LOG = LogManager.getLogger(Main.class.getName());
+    public static Activity activity = new ReadRules();
+    public static Applicant applicant;
+
+    public static void main(String[] args) throws Exception {
+        LOG.info("Program start");
+        applicant = new Applicant();
+        System.out.println("\nWELCOME TO THE UNIVERSITY OF ARTS!\nPlease read the procedure for conducting entrance examinations at the University\n");
+        LOG.info("First of all, the Applicant should read about the procedure for conducting entrance examinations at the University");
+        applicant.setActivity(new ReadRules());
+        applicant.saveState();
```

Comment:
still not descriptive. What information do we save, where is it saved, how can i read this information later. 
`applicant.saveInfoToFile('filepath.txt');`

Reply:
Done